src/components/drag-and-drop/draggable-context/__definition__.js

```diff
@@ -23,18 +22,21 @@ let definition = new Definition('draggable-context', DraggableContext, {
   dataVariable: 'dndData',
 
   propValues: {
-    tbody: false,
-    children: `<thead>
-    <TableRow as="header">
-      <TableHeader />
-      <TableHeader>Country</TableHeader>
-    </TableRow>
-  </thead>
-  <DraggableContext onDrag={ updateDndData }>
-    <tbody>
-      { buildRows() }
-    </tbody>
-  </DraggableContext>`
+    onDrag: 'updateDndData',
+    children: `
+    <div>
```

Comment:
indentation is wrong on here:
![screen shot 2017-08-18 at 11 37 26](https://user-images.githubusercontent.com/1668080/29455587-9ad420f4-8409-11e7-88e2-d63e99dac5ec.png)